modules/core/src/main/java/org/apache/ignite/internal/processors/cache/transactions/IgniteTxManager.java

```diff
@@ -3190,6 +3198,23 @@ private void collectInfo() {
         }
     }
 
+    /**
+     * Profiles transaction.
+     *
+     * @param tx Transaction.
+     * @param commit {@code True} if transaction commited.
```

Comment:
Typo: flag name should be `committed` or `isCommitted`

Reply:
Fixed.